utils/gen_onnx_mlir.py

```diff
@@ -421,7 +421,22 @@
       } else {
         resultTypes.push_back(mlir::UnrankedTensorType::get(
           FloatType::getF32(getContext())));
-      }'''
+      }''',
+  "RandomNormal":
+  '''if (auto attr = dtypeAttr()) {
+      if (dtype() == 0) {
+        resultTypes.push_back(mlir::UnrankedTensorType::get(
+            FloatType::getF16(getContext())));
+      } else if (dtype() == 1) {
+        resultTypes.push_back(mlir::UnrankedTensorType::get(
+            FloatType::getF32(getContext())));
+      } else if (dtype() == 2) {
+        resultTypes.push_back(mlir::UnrankedTensorType::get(
+            FloatType::getF64(getContext())));
+      }
+    }
+    resultTypes.push_back(mlir::UnrankedTensorType::get(
```

Comment:
you have to put an `else` before this so you don't push two result types when the dtype attribute is set, right?